test/language/statements/for/head-const-bound-names-in-stmt.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: The body may not re-declare variables declared in the head
+info: |
+    IterationStatement :
+        for ( LexicalDeclaration Expressionopt ; Expressionopt ) Statement
+
+    It is a Syntax Error if any element of the BoundNames of LexicalDeclaration
+    also occurs in the VarDeclaredNames of Statement.
+negative: SyntaxError
+id: sec-for-in-and-for-of-statements
+es6id: 13.7
```

Comment:
The IDs for files under `test/language/statements/for` point to the wrong section.